tests/plugins/textwatcher/manual/throttle.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: 4.10.0, feature, 1997
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, basicstyles, textwatcher
+
+1. Focus the editor.
+1. Start typing `a` constantly.
+1. Check log above the editor. 
+
+## Expected
+
+Typed text should be logged:
+1. Immediately after first typed character.
+1. After 2000ms intervals.
```

Comment:
Technically it should be something like _not more often than once every 2000ms_. Or something.

Reply:
I think logging the time (including millisecs) would be very helpful for this test.